app/src/main/res/values/strings.xml

```diff
@@ -88,11 +93,7 @@
     ]]>
     </string>
 
-    <string name="error_connect_title">@string/error_connectionfailure_title</string>
```

Comment:
Why do these even exist? Why not just reference the source strings directly? Looks like ahunt did this a while ago...

Reply:
Yeah, I don't know. When I first saw this blocking the new L10N tooling I thought about just removing them. But then I decided to just move them (in Focus we also have a string array).